src/main/proto/tech/figure/servicing/v1beta1/loan_state.proto

```diff
@@ -74,34 +75,35 @@ message LoanState {
-  tech.figure.util.v1beta1.Money      deferred_principal       = 30; // Deferred Principal balance
+  tech.figure.util.v1beta1.Status     loan_status = 7;
+  tech.figure.util.v1beta1.Money      current_p_and_i = 8; // Currently monthly loan payment amount including principal and interest
+  tech.figure.util.v1beta1.Rate       current_interest_rate = 9; // The current interest rate used to calculate the principal and interest payment
+  tech.figure.util.v1beta1.Date       int_accrual_start_date = 10; // The date on which interest accrual began
+  tech.figure.util.v1beta1.Date       int_paid_through_date = 11; // The date through which interest is paid with the current payment. The effective date from which interest will be calculated for the application of the next payment.
+  tech.figure.util.v1beta1.Date       maturity_date = 12; // The date when a loan reaches maturity
+  tech.figure.util.v1beta1.Date       next_due_date = 13; // Date of next due payment for loan
+  int32                               days_delinquent = 14; // Number of days delinquent
+  int32                               remaining_term_months = 15; // Remaining loan term in months
+  tech.figure.util.v1beta1.Rate       servicing_fee_rate = 16; // The fee paid to the servicer, stated as a percentage of the outstanding loan balance
+  tech.figure.util.v1beta1.Rate       apr = 17; // Annualized Percentage Rate of the loan
+  bool                                autopay_flag = 18; // If true, payments are automatically deducted from borrower's financial account
+  tech.figure.util.v1beta1.Money      deferred_int_balance = 19; // Accumulated Interest balance since forbearance_begin_date
+  tech.figure.util.v1beta1.Date       delinquent_date = 20; // If delinquent, date first payment was missed that made the loan currently delinquent
+  uint32                              days_forbearance = 21; // Days elapsed since entering Forbearance
+  tech.figure.util.v1beta1.Date       forbearance_begin_date = 22; // Date forbearance period begins
+  tech.figure.util.v1beta1.Date       forbearance_end_date = 23; // Date forbearance period ends
+  tech.figure.util.v1beta1.Money      interest_paid = 24; // Total interest paid to-date
+  tech.figure.util.v1beta1.Rate       interest_rate_cap = 25; // Interest rate cap
+  tech.figure.util.v1beta1.Date       last_payment_date = 26; // Last payment received date
+  tech.figure.util.v1beta1.Money      monthly_payment_amount = 27; // The minimum monthly payment amount that borrower needs to repay on the loan
+  tech.figure.util.v1beta1.Money      principal_paid = 28; // Sum of amount paid to principal
+  tech.figure.util.v1beta1.Money      past_due_amount = 29; // Amount past due
+  tech.figure.util.v1beta1.Money      deferred_principal = 30; // Deferred Principal balance
+  repeated tech.figure.servicing.v1beta1.ledger                ledger_entry = 70; // Ledger Entries effective on this date
```

Comment:
May I suggest undoing the whitespace-only changes to this file? I see no reason to unalign the field numbers for readability.

Reply:
That's fair!